projects/sidechain-connector-service-typescript/build.sh

```diff
@@ -0,0 +1,3 @@
+npm link ../common-library-typescript
```

Comment:
@idandash could you decorate the bash files according to what I've added last week? I know that you've added these prior to my change, but this way we could stay in-sync.